src/core/AppWrapper/AppWrapper.tsx

```diff
@@ -14,50 +12,14 @@ import "@demo/theme/appStyles.scss";
 import "react-tooltip/dist/react-tooltip.css";
 import "mapbox-gl/dist/mapbox-gl.css";
 
-const {
-	ROUTES: { DEMO }
-} = appConfig;
-
 interface AppWrapperProps {
 	children?: ReactNode;
 }
 
-const AppWrapper: FC<AppWrapperProps> = ({ children }) => {
-	const { identityPoolId, region, userPoolId, userPoolClientId, userDomain, configureAmplify } = useAmplifyAuth();
-
-	const amplifyConfig = useMemo(
-		() => ({
-			Auth:
-				userPoolId && userPoolClientId && userDomain
-					? {
-							identityPoolId,
-							region,
-							userPoolId,
-							userPoolWebClientId: userPoolClientId,
-							oauth: {
-								domain: userDomain,
-								scope: ["email", "openid", "profile"],
-								redirectSignIn: `${window.location.origin}${DEMO}`,
-								redirectSignOut: `${window.location.origin}${DEMO}?sign_out=true`,
-								responseType: "code"
-							}
-					  }
-					: {
-							identityPoolId,
-							region
-					  }
-		}),
-		[identityPoolId, region, userPoolId, userPoolClientId, userDomain]
-	);
-
-	/* Configure Auth and Geo via amplify */
-	configureAmplify(amplifyConfig);
-
-	return (
-		<ThemeProvider theme={appTheme} nonce="dAnIsRazNonCe">
-			{children}
-		</ThemeProvider>
-	);
-};
+const AppWrapper: FC<AppWrapperProps> = ({ children }) => (
+	<ThemeProvider theme={appTheme} nonce="dAnIsRazNonCe">
```

Comment:
Isn't the nonce supposed to be randomly generated on every request?

Reply:
It should be, I'll look into this. This was actually worked on by another developer at the start of this project. Even if we remove this, we will be good. I'll give that a test and let you know.